Docs: Add and clarify query string handling tips (#3319)
fastify/fastify: docs/Server.md

Please note that setting this option to `false` goes against
[RFC3986](https://tools.ietf.org/html/rfc3986#section-6.2.2.1).

> Also note, this setting will in no way affect query strings. If you want to change the way query strings are handled take a look at [querystringParser](./Server.md#querystringParser).

Reviewer: ```suggestion
Also note, this setting will in no way affect query strings. If you want to change the way query strings are handled take a look at [`querystringParser`](./Server.md#querystringParser).
```

> Also note, this setting will in no way affect query strings. If you want to change the way query strings are handled take a look at [`querystringParser`](./Server.md#querystringParser).

Reviewer 2: ```suggestion
Also note, this setting will not affect query strings. If you want to change the way query strings are handled take a look at [`querystringParser`](./Server.md#querystringParser).
```

})
```

> You could also stick with the parser that Fastify uses but change some handling behaviour, like the example below for case insensitive keys and values.

Reviewer 2: ```suggestion
You can also use Fastify's default parser but change some handling behaviour, like the example below for case insensitive keys and values:
```
